src/nextDay/index.ts

```diff
@@ -0,0 +1,56 @@
+import requiredArgs from '../_lib/requiredArgs/index'
+import getDay from '../getDay'
+import addDays from '../addDays'
+import toDate from '../toDate'
+import isValid from '../isValid/index'
+
+const baseMap = [7, 6, 5, 4, 3, 2, 1]
+
+/**
+ * @name nextDay
+ * @category Weekday Helpers
+ * @summary When is the next day of Week?
+ *
+ * @description
+ * When is the next day of the week? 0-6 the day of the week, 0 represents Sunday.
+ *
+ * @param {Date | number} date - the date to check
+ * @param {number} day - day of week
+ * @returns {Date} the date is the next day of week
+ * @throws {TypeError} 2 arguments required
+ *
+ * @example
+ * When is the next Monday after Mar, 20, 2020?
+ * const result = nextDay(new Date(2020, 2, 20), 1)
+ * => new Date(2020, 2, 23)
+ *
+ * @example
+ * When is the next Tuesday after Mar, 21, 2020?
+ * const result = nextDay(new Date(2020, 2, 21), 2)
+ * => new Date(2020, 2, 24)
+ **/
+
+export default function nextDay(date: Date | number, day: number): Date {
```

Comment:
Let's instead of checking the type in runtime and throwing `throw new RangeError('Invalid Day')` use an alias for day: 
```suggestion
export default function nextDay(date: Date | number, day: Day): Date {
```

...and add to `types.ts`:

```ts
export type Day = 0 | 1 | 2 | 3 | 4 | 5 | 6
```

Reply:
Done